tests/test_immutable_morphology.cpp

```diff
@@ -0,0 +1,356 @@
+#include "contrib/catch.hpp"
+#include <cmath>
+
+#include <morphio/endoplasmic_reticulum.h>
+#include <morphio/glial_cell.h>
+#include <morphio/mito_section.h>
+#include <morphio/mitochondria.h>
+#include <morphio/morphology.h>
+#include <morphio/mut/morphology.h>
+#include <morphio/properties.h>
+#include <morphio/section.h>
+#include <morphio/soma.h>
+
+
+namespace {
```

Comment:
just for my knowledge : in this particular case using static or an anonymous namespace is identical no ?

Reply:
yeah; anonymous namespace is more `C++` ish; I still do `static` sometimes when I'm destracted.